autopush/web/base.py

```diff
@@ -140,7 +140,8 @@ class Notification(object):
 
     @property
     def data_length(self):
-        return len(self.data or "")
+        # Not sure why the following is not being picked up by cover.
+        return len(self.data or "")  # pragma: nocover
```

Comment:
because this class is that old simplepush Notification type that we don't even need any more and you get to kill the entire thing! :feelsgood:

Reply:
Ah, and apparently I missed killing the `autopush/web/simplepush/py` file. That's why I was still finding references. 🤡